app/javascript/onboarding/components/BioForm.jsx

```diff
@@ -51,21 +51,29 @@ class BioForm extends Component {
     const { prev } = this.props;
     return (
       <div className="onboarding-main">
+        <Navigation prev={prev} next={this.onSubmit} />
         <div className="onboarding-content about">
-          <h2>About You!</h2>
-          <form>
-            <label htmlFor="summary">
+          <header className="onboarding-content-header">
+            <h1 className="title">
+              Introduce yourself
+            </h1>
+            <h3 className="subtitle">
               Tell the community about yourself! Write a quick bio about what
               you do, what you&apos;re interested in, or anything else!
```

Comment:
This is non-blocking and is me just being a bit nitpicky, but I feel like there are one too many exclamation points here. I think it might read better if we kept the first exclamation point and removed the second. Thoughts? 😄

Reply:
@juliannatetreault good points — I agree. I'll make a quick modification to this copy to make it less intense.

We will end up re-doing this copy here in P2 when we combine the `BioForm` with the `PersonalInfo` form so I think we will read better in that case as well.

Reply:
Okay, awesome! 👍 👍